src/VehicleSetup/JoystickConfig.qml

```diff
@@ -453,6 +453,14 @@ SetupPage {
 
                                     onClicked: _activeJoystick.throttleMode = 1
                                 }
+
+                                QGCCheckBox {
```

Comment:
Should the visibility of this option depend on https://github.com/mavlink/qgroundcontrol/pull/5185/files#diff-4dd9fb0da73f9007de515d2da5b6dbfdR422?